pom.xml

```diff
@@ -84,9 +84,9 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>org.hibernate</groupId>
+            <groupId>org.hibernate.validator</groupId>
             <artifactId>hibernate-validator</artifactId>
-            <version>6.2.0.CR1</version>
+            <version>7.0.3.Final</version>
```

Comment:
This is not the version in the server is it? Could cause problems in runtime

Reply:
Current docs show that this is the provided version.